packages/graphql/tests/integration/experimental/interfaces-top-level.int.test.ts

```diff
@@ -700,6 +704,116 @@ describe("Top-level interface query fields", () => {
         });
     });
 
+    describe("add limit directive", () => {
+        beforeAll(async () => {
+            typeDefs = typeDefs + `extend interface MyInterface @limit(default: 1, max: 3) `;
```

Comment:
I believe that it will be better to separate this test suite into separate files with separated typeDefs, mutate the typedefs here will propagate the side effect around the parent test suite, making the test more likely to become flaky and hard to maintain, in fact, this test suite will propagate effects to the next suite "add schema configuration".

Reply:
The same unnecessary complexity IMO appears when you try to identify the current data set used in the test suite where you have to trust other suites to not mutate the current database state.